plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -2183,4 +2206,17 @@ public void testAggregationPushdownWithArrays()
                         "(56, VARCHAR 'string_8400', BIGINT '1')," +
                         "(1000, VARCHAR 'string1_8401', BIGINT '1')");
     }
+
+    @Test
+    public void testJson()
+    {
+        assertThat(query("SELECT string_col, json_col, updated_at_seconds FROM " + JSON_TYPE_TABLE))
+                .isFullyPushedDown();
```

Comment:
The test purpose is ensuring correctness of json type. Please verify the result and remove `isFullyPushedDown`.